packages/common/src/Logger.ts

```diff
@@ -0,0 +1,12 @@
+
+  /**
+   * Single method to be used for logging purpose.
+   *
+   * @param {any} message Message to be logged
+   */
+   export function _logMessage(message: any) {
+    // Method would be called ifLogsEnabled is true
```

Comment:
Logger will be a class with static method log with a static state `isLogsEnabled`, which is set by the main package constructor based on users input by calling the static method `enableLogs` on the class.

```
export class Logger {
    static isLogsEnabled: boolean = false;

    static log(message: any) {
      if(this.isLogsEnabled && console.log) {
        console.log(message);
      }
    }

    static enableLogs(_isLogsEnabled: boolean) {
      this.isLogsEnabled = _isLogsEnabled;
    }
}

```
Also i don't see the usage of this method anywhere in the code yet.

Reply:
acknowledged and noted for logger ticket scope as discussed